Doc/library/dis.rst

```diff
@@ -210,6 +213,11 @@ operation is being performed, so the intermediate analysis object isn't useful:
    This generator function uses the ``co_firstlineno`` and ``co_lnotab``
    attributes of the code object *code* to find the offsets which are starts of
    lines in the source code.  They are generated as ``(offset, lineno)`` pairs.
+   See ``Objects/lnotab_notes.txt`` for the ``co_lnotab`` format and how to
```

Comment:
Using a source ref here turn this into a suitable hyperlink to GitHub:

    :source:`Objects/lnotab_notes.txt`